### ISSUE TITLE
`yarn jest` TypeError: xxx.default is not a constructor -- default export not output as expected

### ISSUE THREAD

**Issue by liyuanqiu** [jaredpalmer/tsdx#83]
### Current Behavior

`tsdx` yarn test will report error:
```
TypeError: xxx.default is not a constructor
```
when import a class created by `tsdx` and try to instance it.

***reproduction:***
1. create a lib using `tsdx`
```bash
tsdx create mylib-a
```
2. modify index.ts to a `class`
```typescript
// mylib-a/src/index.ts
export default class MyLibA {}
```
3. build
```bash
yarn build
```
4. create another lib using `tsdx` called `mylib-b`
5. move `mylib-a` to `mylib-b/src/`
6. import `mylib-a` in `mylib-b/test/blah.test.ts`
```typescript
import MyLibA from '../src/mylib-a';
// this line is error when `yarn test`
// TypeError: xxx.default is not a constructor
const instance = new MyLibA();
```
7. yarn test
---
If I don't use `default` to export class:
```typescript
export class MyLibA {}
```
And import like this:
```typescript
import { MyLibA } from ...;
```
Everything will be ok, but it is just a workaround.

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX           | ^0.5.6
| TypeScript       | ^3.4.5
| Browser          | -
| npm/Yarn         | yarn@1.13.0
| Operating System | macOS Mojave version 10.14.4 (18E226)

**Comment by liyuanqiu** [comment]
So if one npm library created by `tsdx` wants to use a class imported from another npm library created by `tsdx` too in it's tests, the tests won't pass.

Sorry my mother tongue is not English, maybe this description is a little bit confusing...

**Comment by iamtheworstdev** [comment]
@jaredpalmer  any chance you'll be looking into this? noticed it's still happening in a personal project I just created, which conerns me because I only heard about TSDX because a customer is trying to run with it.

**Comment by liyuanqiu** [comment]
> @jaredpalmer any chance you'll be looking into this? noticed it's still happening in a personal project I just created, which conerns me because I only heard about TSDX because a customer is trying to run with it.

My solution is NOT to use "Default Exports" and "Hybrid Exports", only to use "Named Exports".

**Comment by agilgur5** [comment]
Sounds like this was probably a duplicate of #165 , which was fixed by my #327
